gatsby-config.js

```diff
@@ -181,12 +183,20 @@ module.exports = {
         //
         // If this is addressed in MDX v2, we can safely remove this.
         remarkPlugins: [],
+        rehypePlugins: [
+          [
+            require('./rehype-plugins/gatsby-inline-images'),
+            { spacing: '0.25rem' },
+          ],
+        ],
         gatsbyRemarkPlugins: [
           {
             resolve: 'gatsby-remark-images',
             options: {
               maxWidth: 1200,
               linkImagesToOriginal: false,
+              backgroundColor: 'transparent',
+              disableBgImageOnAlpha: true,
```

Comment:
what does this option do? `disableBgImageOnAlpha`?

Reply:
According to the [docs](https://www.gatsbyjs.com/plugins/gatsby-remark-images/):

> Images containing transparent pixels around the edges results in images with blurry edges. As a result, these images do not work well with the “blur up” technique used in this plugin. As a workaround to disable background images with blurry edges on images containing transparent pixels, enable this setting.

There were a few of these transparent pngs that looked rather weird when they loaded when using that background image, so I opted to turn that off.